src/main/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/xml/DocumentUtils.kt

```diff
@@ -29,7 +31,23 @@ object DocumentUtils {
         return null
     }
 
-    private fun getOuCode(item: Node): String? {
+    fun getCourtCode(documents: Element): String? {
```

Comment:
Was worried that there was a large overhead for using xPath , especially `XPathFactory.newInstance().newXPath() `so did an implementation without it.